MANIFEST.in

```diff
@@ -5,14 +5,12 @@ include README.md
 include TODO.rst
```

Comment:
Since we are at it, we can probably remove this line too! And maybe remove the file...

Reply:
Fixed cdb57149